frontend/src/v5/ui/routes/userSignup/userSignupForm/userSignupForm.component.tsx

```diff
@@ -203,7 +195,7 @@ export const UserSignupForm = ({ completeRegistration }: UserSignupFormProps) =>
 						<UserSignupFormStepTermsAndSubmit
 							{...getStepProps(2)}
 							formIsSubmitting={formIsSubmitting}
-							hasUnexpectedError={hasUnexpectedError}
+							expectedErrorValidators={[usernameAlreadyExists, emailAlreadyExists]}
```

Comment:
I did it because this is the component aware of the errors and in charge of handling them, whereas the `UserSignupFormStepTermsAndSubmit` does not deal with emails and usernames directly. As such, I assumed it should not be aware of possible errors that arise from other steps of the signup form. 
I will update the code to reflect this comment though.